dev-ruby/net-sftp/net-sftp-2.1.2-r3.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-USE_RUBY="ruby23 ruby24 ruby25"
+USE_RUBY="ruby24 ruby25 ruby26"
```

Comment:
if it does not pass tests it can't be added.

Reply:
It does not indeed. But neither does it with any other ruby version.
It looks like the last `net-ssh` version that `net-sftp` passes tests with is 3.2.0, which is not present in the tree currently.
A newer version of `net-sftp` that passes tests with modern `net-ssh` has not been released yet. `net-sftp` v2.1.3.rc3 and v3.0.0.beta1 pass the tests fine with `net-ssh` 5.2.0 and all listed ruby versions, including 2.6. I am not sure if it is OK to bring to rc/betas for gems in the gentoo tree though.

Reply:
Thanks for detailed info, I'll leave decision on this to @graaff 
if ruby2.6 added I'll be more than happy to merge vagrant stuff and comepletely swtich to 2.6 =)

Reply:
It turned out that we had patches backported for that already in net-sftp-2.1.2-r3 but they were not properly applied. This is now fixed in 2.1.2-r4 and that revision also has ruby26 support.

Reply:
Thanks! I rebased the branch ommiting the `dev-ruby/net-sftp` commit, since it is no longer necessary.